src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -1,47 +1,120 @@
 // Copyright 2020 Google LLC
-
+//
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-
+//
 //     https://www.apache.org/licenses/LICENSE-2.0
-
+//
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package com.google.sps.servlets;
 
 import java.io.IOException;
-import javax.servlet.annotation.WebServlet;
-import java.util.Arrays;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Arrays;
```

Comment:
Please, try to keep alphabetical order of imports.
I see that you put `javax.servlet.annotation.WebServlet` in its rightful place, but now `java.util.Arrays` should be moved back up to the second place, before `ArrayList`.

Reply:
Actually I'm wrong, both as per alphabetic order AND as per [style guide](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing) demanding ASCII order, the `ArrayList` comes just before `Arrays`, because capital `L` is before lowercase `s`. So it should be: `ArrayList, Arrays, List`.